src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -1381,6 +1381,14 @@
   </data>
   <data name="DoNotNameEnumValuesReservedTitle" xml:space="preserve">
     <value>Do not name enum values 'Reserved'</value>
+  <data name="DoNotDeclareEventFieldsAsVirtualDescription" xml:space="preserve">
+    <value>Do not declare virtual events in a base class and override them in a derived class. The C# compiler does not handle these correctly and it is unpredictable whether a subscriber to the derived event will actually be subscribing to the base class event.</value>
+  </data>
+  <data name="DoNotDeclareEventFieldsAsVirtualMessage" xml:space="preserve">
+    <value>Do not declare event fields as virtual.</value>
```

Comment:
Do you mean to update the message? I see the analyzer does pass in the event name as the argument to reported diagnostic now..